manageiq-providers-autosde.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "typhoeus", "~> 1.4"
+  spec.add_dependency "autosde_openapi_client", "~> 1.0"
```

Comment:
:+1: nice there you go

Not sure if you're sold on the name but having `openapi` in the gem name probably isn't needed.  Maybe `autosde_client` or `autosde_oas_client` or something like that?

Reply:
@borisko123 I added autosde_openapi_client with fixed version to autosde genspec (so the changes in the gemfile in the core are not needed). please make sure everything is working, and also consider @agrare suggestions on changing the name and using "~> 1.0" for the version (I used the fixed version because I remember you are still working on 1.0.4 and I'm not sure if it will work with it).

Reply:
@agrare please let me remain the name 'autosde_openapi_client' for gem.
1. it is not so simple to change it 
2. the oas means exactly OpenAPI Specification (OAS), which is, in essence, the same
3. oas also means https://en.wikipedia.org/wiki/Organization_of_American_States, may be confused😄

Reply:
Okay I see it is already on rubygems with this name, I don't like it because the fact that it is an openapi client doesn't matter to the caller it is an implementation detail.

Reply:
I agree. Maybe we'll create a differently named gem at a later phase.